relay-server/src/services/buffer/mod.rs

```diff
@@ -243,42 +243,17 @@ impl EnvelopeBufferService {
     /// Wait for the configured amount of time and make sure the project cache is ready to receive.
     async fn ready_to_pop(
         &mut self,
-        partition_tag: &str,
         buffer: &PolymorphicEnvelopeBuffer,
         dequeue: bool,
     ) -> Option<Permit<legacy::DequeuedEnvelope>> {
-        relay_statsd::metric!(
-            counter(RelayCounters::BufferReadyToPop) += 1,
```

Comment:
We added this metric to debug a specific problem in production (see where `ready_to_pop` got cancelled).